ProjFS.Mac/PrjFSKext/PrjFSKext/PrjFSLogUserClient.hpp

```diff
@@ -26,5 +26,18 @@ class PrjFSLogUserClient : public IOUserClient
     virtual IOReturn clientMemoryForType(UInt32 type, IOOptionBits* options, IOMemoryDescriptor** memory) override;
     virtual IOReturn registerNotificationPort(mach_port_t port, UInt32 type, io_user_reference_t refCon) override;
     
+    virtual IOReturn externalMethod(
+        uint32_t selector,
+        IOExternalMethodArguments* arguments,
+        IOExternalMethodDispatch* dispatch = 0,
+        OSObject* target = 0,
+        void* reference = 0) override;
```

Comment:
Minor: Set default pointer values to `nullptr` rather than 0

Reply:
Fair point, those were copied from the declaration of the overridden function. I've fixed them up now and also fixed up the equivalent issue in the provider user client class.